src/components/password/password.test.tsx

```diff
@@ -0,0 +1,188 @@
+import React from "react";
+import { render, screen } from "@testing-library/react";
+import userEvent from "@testing-library/user-event";
+import Password from "./password.component";
+import guid from "../../__internal__/utils/helpers/guid";
+
+const mockedGuid = "guid-12345";
+jest.mock("../../__internal__/utils/helpers/guid");
+
+(guid as jest.MockedFunction<typeof guid>).mockImplementation(() => mockedGuid);
+
+test("should render the `Textbox` component as a child on render", () => {
```

Comment:
**suggestion(non-blocking)**: since the `Textbox` component being a child is an implementation detail, can we assert that the input box and the 'Show Password' button are visible instead?

I think checking these two elements only should be sufficient since they are what the user will actually notice.

Reply:
Yeah, i'm happy with that. I can change the text descriptions referencing `Textbox` to just reference the input instead